docs/DeveloperGuide.md

```diff
@@ -191,6 +191,30 @@ The notdone mechanism is similar to that of the done mechanism, except that the
 
 <div style="page-break-after: always;"></div>
 
+### Workload Distribution feature
+
+#### Implementation
+
+The workload distribution feature is implemented by `ModuleManager`. It uses HashMap and an OberservableList to monitor the workload of each module and PieChart in javaFX to visualizes it. More explicitly, it implements by 4 HashMaps using Module as key. The first three maps are used in the ModuleList while the map of weighted sum is used in the PieChart.
```

Comment:
I think instead of "it implements by 4 HashMaps", it should be "it is implemented by 4 HashMaps".

Reply:
OK!